presto-docs/src/main/sphinx/release/release-0.266.rst

```diff
@@ -0,0 +1,39 @@
+=============
+Release 0.266
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix resource group concurrency in multi coordinator to not allow more than configured queries to run.
```

Comment:
Is multi-coordinator out of beta - @tdcmeehan  ?

Reply:
Let's remove this